modules/apps/consent-management-platform/consent-management-platform-web/bnd.bnd

```diff
@@ -0,0 +1,3 @@
+Bundle-Name: Liferay Consent Management Platform Web
+Bundle-SymbolicName: com.liferay.consent.management.platform.web
+Bundle-Version: 1.0.0
```

Comment:
It is not a real web module. It is not serving any resource. Usually web modules declare the `Web-ContextPath` header